docs/source-app/workflows/byoc/index.rst

```diff
@@ -0,0 +1,92 @@
+
+#################################
+Run Apps on your own cloud (BYOC)
+#################################
+
+**Audience:** Users looking to run Lightning Apps on their own private cloud.
+
+----
+
+*******************
+A bit of background
+*******************
+
+BYOC - Bring Your Own Cloud, is an alternate deployment model to Lightning Cloud (fully managed SaaS).
+BYOC separates the control and data plane. The data plane, that includes
+Lightning clusters, services and Lightning Apps, reside inside the user’s VPC.
+The control plane resides on Lightning Cloud.
+
+Setup begins with configuring a cloud provider (today AWS, but more are coming soon) with credentials for
+delegated access and an identity provider for secure access to the data plane.
+
+Next, as part of the environment creation process, users configure networking,
+security, and select Lightning cluster configuration options based on their use cases.
+
+After submitting the request,the Lightning control plane creates the required cloud infrastructure
+and sets up the environment with the Lightning cluster along with a Lightning Kubernetes operator.
+
+
+*******************************
+Create a Lightning BYOC cluster
+*******************************
+
+You must have your cloud configured before you try and create a BYOC cluster.
+
+Create a Lightning BYOC cluster using the following command:
+
+PLEASE PROVIDE AN EXAMPLE WITH CLOUD PROVIDER PARAMETERS
```

Comment:
@edenlightning

Reply:
here's an example request to create a cluster:

```
lightning create cluster test-raphael9 --provider aws --role-arn arn:aws:iam::1234567890:role/lai-byoc --external-id dummy --region us-west-2 --instance-types t3.xlarge --cost-savings
```

a full list of parameters is available [here](https://github.com/Lightning-AI/lightning/blob/master/src/lightning_app/cli/lightning_cli_create.py#L13)

I think we should link to our [terraform bootstrap](https://github.com/Lightning-AI/terraform-aws-lightning-byoc) repository as well - just need to make it public first.

Reply:
Added something for the Terraform module. Please take a peek